chains/lightlink-goerli-testnet.json

```
@@ -0,0 +1,25 @@
+{
+  "alias": "lightlink-goerli-testnet",
+  "blockTimeMs": 502,
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": true
+      },
+      "url": "https://pegasus.lightlink.io/api"
+    },
+    "browserUrl": "https://pegasus.lightlink.io/"
+  },
+  "id": "1891",
+  "name": "LightLink Goerli Testnet",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://replicator.pegasus.lightlink.io/rpc/v1"
+    }
+  ],
+  "skipProviderCheck": false,
```

I think this is the wrong way around 😬 

```suggestion
  "skipProviderCheck": true,
```

---

oh, my bad. fixing it